homeassistant/components/modbus/strings.json

```diff
@@ -68,5 +68,11 @@
         }
       }
     }
+  },
+  "issues": {
+    "deprecated_close_comm_config": {
+      "title": "\"close_comm_on_error:\" is deprecated",
+      "description": "Please remove parameter \"close_comm_on_error:\" from the modus entry in configuration.yaml.\nCommunication is automatically closed on errors. Please review the other parameters for error handling."
```

Comment:
What other parameters? Note, I am a modbus noob so I don't know all the specific terms. But this doesn't point me in a direction what to solve.

Maybe if you explain it to me I can help create a good mesasge. You can also use URLs and stuff to point to specific docs.

Reply:
Problem is that there are a number of parameters, and if you do not know modbus, you do not know how to use them....most of the users refer to the forum and see what other users did. 

I did not mention the other parameters on purpose, before you asked, because in my experience they confuse users who do not know the protocol.

"Configuring modbus communication" in https://www.home-assistant.io/integrations/modbus/ tries to explain it.

the relevant ones are "delay", "message_wait_milliseconds", "retries", "retry_on_empty" and "timeout"

But the entities themself also have relevant parameters, like "lazy_error_count".

I think the best way is to ask the user to remove this parameters, because it happens automatically, and have a reference to the documentation with the other parameters.

I appreciate help in making a good message....that is a couple of factors more difficult than programming :-)